### DISCUSSION TITLE
Mise ignore a single missing tool

### DISCUSSION THREAD

**Discussion by drewbitt** [jdx/mise#1576] [Q&A] [answered]
Hi,

For example, in node, if I have a .nvmrc with 'v18', mise will say 'mise missing: node@18.19.0'. I use mise for other packages but I don't want to use it for node. I use other tooling for just node, and although I do have node@18.19.0 as my default node, it will still show this error. How can I hide the 'missing' check for just the node package?

`node   system`or `nodejs    system` in .tool-versions did not work, unlike in asdf

**Comment by jdx** [comment, marked as answer]
https://mise.jdx.dev/configuration.html#disable-tools

**Comment by drewbitt** [comment]
Thanks!

```
[settings]
disable_tools = ['node']
```
